### DISCUSSION TITLE
"Stop", the time line slider, and the screen

### DISCUSSION THREAD

**Discussion by LarryM04** [smplayer-dev/smplayer#495] [Q&A]
I had to reload my Ubuntu PC and now that I have SMPlayer is behaving differently.  Used to be when I'd click the "Stop" button twice, the screen would go black.  Now the screen stays on the whatever was displayed when I first clicked.  I've tried with mpv and mplayer.  I get the correct behavior with mpv, but then if I advance the time line slider, the screen goes blank and nothing seems to get the display to return.  With mplayer, the time line slider works, but then the "Stop" button doesn't blank the screen.

I've combed thru the preferences and don' see anything that would seem to help this.

Any clues would be greatly appreciated.

**Comment by smplayer-dev** [comment]
Try changing the video output in preferences -> general -> video.

**Reply by LarryM04** [reply]
Okay, that might be the answer, but now when I have mpv selected it pops up in its own window despite having Preferences->Advanced->MPlayer/mpv "Run MPlayer/mpv in its own window" NOT checked.  I justed checked that option to see it it was reversed, but no when checked mpv is really seperate.  When not checked, the mpv isn't completely within SMPlayer's area, it has a title bar with the name of the video being played, and isn't centered in the SMPlayer window.  ???

**Reply by smplayer-dev** [reply]
Try another video output.
